src/components/Note.js

```diff
@@ -0,0 +1,60 @@
+import useMediaQuery from '../utils/useMediaQuery'
+import styles from '../css/Note.module.css'
+
+const Note = ({ children, primaryColor, secondaryColor }) => {
+  const isDesktop = useMediaQuery('(min-width: 600px)')
+
+  if (!primaryColor) {
+    primaryColor = '#3D73C7'
+  }
+
+  if (!secondaryColor) {
+    secondaryColor = '#2B5699'
+  }
+
+  return (
+    <div
+      className={styles.container}
+      style={{
+        '--primary-color': primaryColor,
+        '--secondary-color': secondaryColor
+      }}
+    >
+      <div className={styles.main}>{children}</div>
+      <div className={styles.pageCurl}>
+        {isDesktop ? (
+          <svg
+            className={styles.borderEdge}
+            width="73"
+            height="54"
+            viewBox="0 0 73 54"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              d="M72.2627 0.610107L0.262695 53.6101C3.35196 39.8428 4.147 32.1699 0.262695 15.1101C29.4332 16.0221 45.0596 11.9848 72.2627 0.610107Z"
+              fill={secondaryColor}
+            />
+          </svg>
```

Comment:
You don't need to inline the whole svg. You can import it's react comp, then style it from that.
so in your svg file if you have
```xml
          <svg
            width="73"
            height="54"
            viewBox="0 0 73 54"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M72.2627 0.610107L0.262695 53.6101C3.35196 39.8428 4.147 32.1699 0.262695 15.1101C29.4332 16.0221 45.0596 11.9848 72.2627 0.610107Z"
            />
          </svg>
```

then in react you can have
```javascript
import { ReactComponent as MySVG } from '../path/to/my.svg'
...
<MySVG fill={secondaryColor} className={styles.borderEdge} /> 
```

But going even further, because the parent has the --secondary-color, you can not assign the fill at all, and then just do it in the `.borderEdge` class in .css
```css
.borderEdge {
  ...
  fill: var(--secondary-color);
}
```